test/fixtures/dialects/db2fori/create_table_field_name_with_pound_sign.yml

```diff
@@ -0,0 +1,49 @@
+# YML test files are auto-generated from SQL files and should not be edited by
+# hand. To help enforce this, the "hash" field in the file must match a hash
+# computed by SQLFluff when running the tests. Please run
+# `python test/generate_parse_fixture_yml.py`  to generate them after adding or
+# altering SQL files.
+
+
+# ::TODO:: run python test/generate_parse_fixture_yml.py
+# ::TODO:: figure out why when I run the command above I get the exception:
```

Comment:
I don't get that exception on your branch and it looks fine to me.

Are you definitely using your local copy (e.g. `pip install -e .`)?
Or sometimes better to use tox:
```
tox -e generate-fixture-yml
```

Reply:
Thanks! The tox command worked well  - https://github.com/sqlfluff/sqlfluff/pull/3231/commits/7fbacc85bc5a6a7dd7d2cf03fded6deb013b7ec5